pkg/resourcecreator/poddisruptionbudget.go

```diff
@@ -0,0 +1,37 @@
+package resourcecreator
+
+import (
+	nais_io_v1alpha1 "github.com/nais/liberator/pkg/apis/nais.io/v1alpha1"
+	policyv1beta1 "k8s.io/api/policy/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+)
+
+func PodDisruptionBudget(app *nais_io_v1alpha1.Application) *policyv1beta1.PodDisruptionBudget {
+	if app.Spec.Replicas.Max == 1 {
+		return nil
+	}
+
+	if app.Spec.Replicas.Min == app.Spec.Replicas.Max {
```

Comment:
Hvorfor ingen PDB hvis man har like mange min som max?

Reply:
Fordi hvis man har satt eksplisitte like verdier her kan det bety at appen ikke tåler å kjøre med 1 ekstra pod.

Forsøk på å unngå å brekke noe, mulig det er litt vel defensivt.

Reply:
Hum, høres ut som noe vi ikke burde anta, men forhøre oss om.

Reply:
Tenkt litt mer på det, min == max er ikke en garanti vi kan gi uansett, så jeg fjerner det i steden for.